bitnami/jupyterhub/templates/hub/deployment.yaml

```diff
@@ -75,7 +75,6 @@ spec:
       terminationGracePeriodSeconds: {{ .Values.hub.terminationGracePeriodSeconds }}
       {{- end }}
       initContainers:
-        {{- if .Values.postgresql.enabled }}
```

Comment:
Shouldn't we still have a flag for enabling the external database's initContainers?

Reply:
postgresql.enabled AFAIK is not a flag that has anything to do with external databases it is the flag for enabling the bundled postgresql. https://github.com/bitnami/charts/blob/aea4c612bc0c1816e7d0380263691fc8f21f19c2/bitnami/jupyterhub/values.yaml#L1748
and https://github.com/bitnami/charts/blob/aea4c612bc0c1816e7d0380263691fc8f21f19c2/bitnami/jupyterhub/Chart.yaml#L18-L22

So basically at some point someone got confused, and instead of the flag being specifically for enabling the internal db as per its description they guarded a section of code that was also meant to handle external dbs. I removed the guard, because it was contradictory and effectively meant you could never enable an external database without the internal one, which would mess up the tertiaries anyway.

There also is no external database flag only an internal one:
https://github.com/bitnami/charts/blob/aea4c612bc0c1816e7d0380263691fc8f21f19c2/bitnami/jupyterhub/values.yaml#L1756-L1785

 It simply would not work with this guard, as it is the source of the problem itself, which caused empty init containers leading to a broken sqlite install.